plugins/parsers/json_v2/parser.go

```diff
@@ -113,6 +117,8 @@ func (p *Parser) Init() error {
 			p.Configs[i].Location = loc
 		}
 	}
+	p.Reset()
+
```

Comment:
Doesn't seem necessary in Init?

Reply:
No, not strictly necessary, but one can think of Reset() as implementing ensure pattern, so it is logical to place it as the last operation in `Init()`.